static/js/interactive-map/PinClusterer/PinClusterer.js

```diff
@@ -4,6 +4,33 @@ import { PinOptions } from '../Maps/MapPin.js';
 import { PinProperties } from '../Maps/PinProperties.js';
 import { Type, assertType, assertInstance } from '../Util/Assertions.js';
 
+/**
+ * Represents a cluster of {@link MapPin}s on a map.
+ * @member {MapPin} clusterPin
+ * @member {MapPin[]} pins
+ * @see {PinCluster}
```

Comment:
do we need an '@see' here if the `PInCluster` class is right below it?

Reply:
Deferring! This is coming directly from a Consulting commit. Adding to the description of SLAP-1094